### PR TITLE
Added AboutFragment test cases

### LINKED ISSUE #2586: [Code health] AboutFragment Test Cases
Write AboutFragment Test Cases

### PR DESCRIPTION
**Pull request by anandwana001**
Fixes #2586 

Part 1 of the AboutFragment Test coverage

@shobhitagarwal1612 @sufyanAbbasi  PTAL?

Screenshot 2024-07-29 at 4 52 43 PM

### CODE REVIEW

File: ground/src/main/java/com/google/android/ground/ui/home/AboutFragment.kt [hunk 1]
```diff
@@ -72,7 +73,8 @@ class AboutFragment : AbstractFragment() {
         Image(
           bitmap = R.drawable.ground_logo.toImageBitmap(),
           contentDescription = "Logo",
-          modifier = Modifier.padding(0.dp, 8.dp, 0.dp, 22.dp).width(122.dp).height(122.dp),
+          modifier =
+            Modifier.testTag("logo").padding(0.dp, 8.dp, 0.dp, 22.dp).width(122.dp).height(122.dp),
```
Comment (shobhitagarwal1612) [inline ground/src/main/java/com/google/android/ground/ui/home/AboutFragment.kt:77]: Can we use contentDescription instead of "testTag"?

Comment (anandwana001) [reply ground/src/main/java/com/google/android/ground/ui/home/AboutFragment.kt]: Done

File: ground/src/test/java/com/google/android/ground/ui/home/AboutFragmentTest.kt [hunk 2]
```diff
@@ -0,0 +1,66 @@
...
+import dagger.hilt.android.testing.HiltAndroidTest
+import org.junit.Before
+import org.junit.Rule
+import org.junit.Test
+import org.junit.runner.RunWith
+import org.robolectric.RobolectricTestRunner
+
+@HiltAndroidTest
+@RunWith(RobolectricTestRunner::class)
+class AboutFragmentTest : BaseHiltTest() {
```
Comment (shobhitagarwal1612) [inline ground/src/test/java/com/google/android/ground/ui/home/AboutFragmentTest.kt:35]: Can we add remaining coverage for this fragment?

Comment (anandwana001) [reply ground/src/test/java/com/google/android/ground/ui/home/AboutFragmentTest.kt:35]: Doing in parts, as mentioned in PR description "Part 1 of the AboutFragment Test coverage"

**Review by shobhitagarwal1612 — APPROVED** [review]

**Pull request merged**